hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -240,11 +250,28 @@ private boolean displayTable(ManagedRocksIterator iterator,
       return displayTable(iterator, dbColumnFamilyDef, out(), schemaV3);
     }
 
+    // If there are no parent directories, create them
+    File dirFile = new File(fileName);
```

Comment:
How about just add suffix (like: x.0, x.1 x.1 ...)on the output file, instead of crate a directory? just like previous design of this PR.